lib/bdev/rbd/Makefile

```diff
@@ -34,7 +34,7 @@
 SPDK_ROOT_DIR := $(abspath $(CURDIR)/../../..)
 include $(SPDK_ROOT_DIR)/mk/spdk.common.mk
 
-CFLAGS += -I$(SPDK_ROOT_DIR)/lib/bdev/
+CFLAGS += $(DPDK_INC) -I$(SPDK_ROOT_DIR)/lib/bdev/
```

Comment:
This shouldn't need to include DPDK now that the functionality has been moved to the env library.